getting-started-guides/.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea/
+
+# .NET build artifacts
+[Bb]in/
+[Oo]bj/
+
+# Java build artifacts
+build/
+.gradle/
```

Comment:
Let's reorganize to this directory structure:

-> getting-started-guides/
  --> java/
    --> instrumented/
    --> uninstrumented/
  --> dotnet/
    --> instrumented/
    --> uninstrumented/